### PR TITLE
Master Fix

### PR DESCRIPTION
**Pull request by IhorBohdan**

### CODE REVIEW

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/finance/fiscalYears/rollover-fiscal-year.cy.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ import NewOrganization from '../../../support/fragments/organizations/newOrganiz
 import NewInvoice from '../../../support/fragments/invoices/newInvoice';
 import Invoices from '../../../support/fragments/invoices/invoices';
 
-describe('ui-finance: Funds', () => {
+describe('ui-finance: Fiscal Year', () => {
```
Comment (re-jas) [inline cypress/e2e/finance/fiscalYears/rollover-fiscal-year.cy.js:20]: 'Users' is defined but never used
So, if you don't need it, please remove

Comment (re-jas) [inline cypress/e2e/finance/fiscalYears/rollover-fiscal-year.cy.js:20]: * Please uncomment or delete after block which starts on line 146 
* line 8: 'Users' is defined but never used 
* line 56: 'user' is assigned a value but never used.

so, if you don't need them, please remove

Comment (re-jas) [reply cypress/e2e/finance/fiscalYears/rollover-fiscal-year.cy.js:20]: periodStart and periodEnd are duplicated

Comment (re-jas) [reply cypress/e2e/finance/fiscalYears/rollover-fiscal-year.cy.js:20]: could you please add a comment why do you need the waiter on 118 line

File: cypress/e2e/invoices/approve-invoice.cy.js [hunk 2]
```diff
@@ -66,12 +66,12 @@ describe('ui-invoices: Approve invoice', () => {
...
     const transactionFactory = new Transaction();
     Invoices.searchByNumber(invoice.invoiceNumber);
-    Helper.selectFromResultsList();
+    Invoices.selectInvoice(invoice.invoiceNumber);
     Invoices.approveInvoice();
     // check transactions after approve
     cy.visit(TopMenu.fundPath);
     Helper.searchByName(defaultFund.name);
-    Helper.selectFromResultsList();
+    Funds.selectFund(defaultFund.name);
```
Comment (re-jas) [inline cypress/e2e/invoices/approve-invoice.cy.js:74]: image
you have eslint problems

File: cypress/e2e/invoices/cancelling-approved-invoices.cy.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ describe('ui-invoices: Cancelling approved invoices', () => {
 
               cy.visit(TopMenu.fundPath);
               Helper.searchByName(defaultFund.name);
-              Helper.selectFromResultsList();
+              Funds.selectFund(defaultFund.name);
```
Comment (re-jas) [inline cypress/e2e/invoices/cancelling-approved-invoices.cy.js:58]: image
problems with eslint and you imported same module two times with different names (Helper SearchHelper) please delete one

File: cypress/e2e/invoices/test-acquisition-unit-for-invoices.cy.js [hunk 4]
```diff
@@ -80,6 +80,6 @@ describe('ui-invoices: create', () => {
     
     cy.login(user.username, user.password, { path:TopMenu.invoicesPath, waiter: Invoices.waitLoading });
     Invoices.searchByNumber(invoice.invoiceNumber);
-    FinanceHelp.selectFromResultsList();
+    Invoices.selectInvoice(invoice.invoiceNumber);
```
Comment (re-jas) [inline cypress/e2e/invoices/test-acquisition-unit-for-invoices.cy.js:83]: 'FinanceHelp' is defined but never used. Please delete it if you don't need it

File: cypress/e2e/organizations/test-acquisition-unit-for-organizations.cy.js [hunk 5]
```diff
@@ -38,7 +38,7 @@ describe('ui-organizations: Organizations', () => {
   after(() => {
     cy.loginAsAdmin({ path:TopMenu.organizationsPath, waiter: Organizations.waitLoading });
     Organizations.searchByParameters('Name', organization.name);
-    FinanceHelp.selectFromResultsList();
+    Organizations.selectOrganization(organization.name);
```
Comment (re-jas) [inline cypress/e2e/organizations/test-acquisition-unit-for-organizations.cy.js:41]: 'FinanceHelp' is defined but never used. Please delete if you don't need it

File: cypress/e2e/organizations/user-integration-organization.cy.js [hunk 6]
```diff
@@ -70,7 +70,7 @@ describe('ui-organizations: EDI convention in Organization Integration', () => {
     // Found and edit created organization
     Organizations.searchByParameters('Name', organization.name);
     Organizations.checkSearchResults(organization);
-    Organizations.chooseOrganizationFromList(organization);
+    Organizations.selectOrganization(organization.name);
```
Comment (re-jas) [inline cypress/e2e/organizations/user-integration-organization.cy.js:73]: you used waiter on 81 line please add a comment to explain why you need it there

File: cypress/support/fragments/organizations/organizations.js [hunk 7]
```diff
@@ -255,4 +255,8 @@ export default {
       Button({ id: 'clickable-delete-organization-confirmation-confirm' }).click()
     ]);
   },
+
+  selectOrganization:(organizationName) => {
```
Comment (re-jas) [inline cypress/support/fragments/organizations/organizations.js:259]: 'organizationName' is already declared in the upper scope on line 22.

**Review by re-jas — APPROVED** [review]

**Pull request merged**